pkg/cmd/openshift-sdn/cmd.go

```diff
@@ -97,6 +102,13 @@ func (sdn *OpenShiftSDN) Run(c *cobra.Command, errout io.Writer, stopCh chan str
 		klog.Fatal(err)
 	}
 
+	// If our CNI config file exists, remove it so that kubelet doesn't think
+	// we're ready yet.
```

Comment:
I mean, we can keep this, but does it do anything? Does oci-cni ever notice the removal of the file?